static/js/lib/posts.js

```diff
@@ -111,37 +114,50 @@ export class PostVotingButtons extends React.Component<*, *> {
     })
   }
 
+  onTogglePopup = async () => {
+    const { popupVisible } = this.state
+    this.setState({
+      popupVisible: !popupVisible
+    })
+  }
+
   render() {
     const { post, className } = this.props
-    const { upvoting } = this.state
+    const { upvoting, popupVisible } = this.state
     const upvoted = post.upvoted !== upvoting
     const upvoteClass = upvoted ? "upvoted" : ""
 
     return (
-      <div className={`upvotes ${className || ""} ${upvoteClass}`}>
-        {userIsAnonymous() ? (
-          <ReactTooltip id="post-upvote-button">
-            {votingTooltipText}
-          </ReactTooltip>
-        ) : null}
-        <button
-          className="upvote-button"
-          onClick={userIsAnonymous() ? null : this.onToggleUpvote}
-          disabled={upvoting}
-          data-tip
-          data-for="post-upvote-button"
-        >
-          <img
-            className="vote-arrow"
-            src={
-              upvoted
-                ? "/static/images/upvote_arrow_on.png"
-                : "/static/images/upvote_arrow.png"
+      <div>
+        <div className={`upvotes ${className || ""} ${upvoteClass}`}>
+          <button
+            className="upvote-button"
+            onClick={
+              userIsAnonymous() ? this.onTogglePopup : this.onToggleUpvote
             }
-            width="13"
+            disabled={upvoting}
+            data-tip
+            data-for="post-upvote-button"
```

Comment:
I think this `data-tip` stuff was for the old tooltip thingy we were using, we should be ok to delete these two props